org.unia.gcolor3.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "org.unia.gcolor3",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "gcolor3",
+
+    "rename-icon": "gcolor3",
+    "rename-desktop-file": "gcolor3.desktop",
+    "rename-appdata-file": "gcolor3.appdata.xml",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+
+        "--filesystem=xdg-config/gcolor3/config.ini:ro"
+    ],
+    
+    "build-options": {
+        "cflags": "-O2 -g"
```

Comment:
Most, if not all of the flathub builds do:
https://github.com/search?q=org%3Aflathub+-g&type=Code

I'm not 100% sure why they agreed to this since flathub only accepts stable builds so it's not like you would be debugging a broken build.

Reply:
Strange, but if they all do it, let's do so too.